rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2499,6 +2500,17 @@ public long getActiveFederationCreationBlockHeight() {
         return activeFederationCreationBlockHeightOpt.orElse(0L);
     }
 
+    protected void checktimeLockExpiration() {
+        if (activations.isActive(RSKIP264)) {
+            long height = provider.getNextUtxoExpirationCheckpointHeight().orElse(0L);
+            long timestamp = provider.getNextUtxoExpirationCheckpointTimestamp().orElse(0L);
```

Comment:
```suggestion
            long nextUtxoExpirationCheckpointTimestamp = provider.getNextUtxoExpirationCheckpointTimestamp().orElse(0L);
```
Feel free to use other names, but let's try to use something meaningful

Reply:
Resolved